packages/conductor/tasks/lock_s3_file.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "lock_s3_file",
+  "retryCount": 10,
+  "retryLogic": "EXPONENTIAL_BACKOFF",
+  "retryDelaySeconds": 10,
+  "timeoutSeconds": 30,
+  "responseTimeoutSeconds": 30,
+  "timeoutPolicy": "RETRY",
+  "inputKeys": ["fileName", "bucketId", "lockId"],
+  "outputKeys": []
```

Comment:
Don't we need to define `lockState` here as it's outputted when the task is completed?

Reply:
doh!